hw/ip/keymgr_dpe/data/keymgr_dpe.hjson

```diff
@@ -125,6 +125,16 @@
       local:     "false",
       expose:    "true",
     }
+    { name:      "FlopToKmac",
+      desc:      '''
+        If true, insert flops to break a potential long chain between
+        the active key slot and KMAC.
+      ''',
+      type:      "bit",
+      default:   "0",
+      local:     "false",
```

Comment:
@rswarbrick Can you make that a local parameter such that the HJSON is the SSOT and it cannot be overwritten by the instantiation?